### ISSUE TITLE
Add section in Understanding of WCAG2,2:2.5.8 for vertical menus

### ISSUE THREAD

**Issue by ljoakley** [w3c/wcag#3749]
The examples in the Understanding doc for 2.5.8 need to have a section added that addresses spacing in vertical menus without borders. The understanding doesn't address if the 24px requirement can include the padding around the menu text. An example of a vertical menu with no borders around the menu items:

![Image](https://github.com/w3c/wcag/assets/32885548/e17322b5-601b-4214-9b6b-9b56637f09a3)

As an example, if a compute is done using the inspect tool in the developer's toolbar on a menu item in a vertical menu, the result is this: 

![Image](https://github.com/w3c/wcag/assets/32885548/f241e8fe-f79b-489a-a5ea-da4949a0850d)

The menu text is only 15px tall, the smallest box, but the padding adds another 9 pixels for a total of 24px, is this a pass? The total height for this menu item is 24px. The Understanding doc only talks about items with defined borders.

**Comment by patrickhlauke** [comment]
> **target**
> region of the display that will accept a pointer action, such as the interactive area of a user interface component

as the padding of a clickable link/button/control also accepts clicks/taps/pointer actions, this is already covered?

> The Understanding doc only talks about items with defined borders.

not really. https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html might only *show* example figures with borders, but doesn't say anything about actual borders

**Comment by ljoakley** [comment]
In this particular case that I found, the padding as defined in the image above does not accept a pointer action. It is possible for the user to click in between the link text and no action is performed. A screen reader will read all the links in the vertical menu and keyboard actions, e.g. using the arrow keys to move in the menu, work correctly. If a person using a mouse has tremors hitting this very small text target will be very difficult. Other applications include the padding in their link thus making the target larger. 

![Image](https://github.com/w3c/wcag/assets/32885548/2229968b-bb73-4140-ad01-1c53348526b0)

I would say that the target, which is only the text, would be a failure.

**Comment by patrickhlauke** [comment]
> the padding as defined in the image above does not accept a pointer action

so it's not padding of the clickable area, but margin?

**Comment by JAWS-test** [comment]
@ljoakley With 2.5.8, the click area does not matter, but the distance between the click areas is relevant. The click area could be 1x1 px and can fulfill 2.5.8 if the distance to other click areas is sufficiently large. A 1x1 px click area is difficult to hit and if you want better accessibility requirements here, you have to test level AAA with SC 2.5.5

**Comment by ljoakley** [comment]
@JAWS-test I would have to disagree: The size of the [target](https://www.w3.org/TR/WCAG2/#dfn-targets) for [pointer inputs](https://www.w3.org/TR/WCAG2/#dfn-pointer-inputs) is at least 24 by 24 [CSS pixels](https://www.w3.org/TR/WCAG2/#dfn-css-pixels), except where:

Spacing: Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the [bounding box](https://www.w3.org/TR/WCAG2/#dfn-bounding-boxes) of each, the circles do not intersect another target or the circle for another undersized target;

According to the minimum standard, placing the 24 CSS pixel circle on a link in the first menu image will intersect with both the menu item above and below it. The Understanding doc does not speak to links that have no border, all the examples show links with borders. I would add an example of a vertical menu with no borders both passing the minimum standard and failing the standard.

**Comment by JAWS-test** [comment]
@ljoakley I don't really understand what your question or suggestion for improvement is. I think SC 2.5.8 is very clear in terms of vertical menus. If you think there is a case where it is different, it would be good to have a real example and not just screenshots. A real example doesn't have to be a URL to an existing page, just some CSS and HTML code is enough

**Comment by mbgower** [comment]
> According to the minimum standard, placing the 24 CSS pixel circle on a link in the first menu image will intersect with both the menu item above and below it. 

To make sure I'm completely understanding the situation here, there is a 15px high target, then 9px of  non-interactive padding, then another 15px high target. So, we have undersized targets. Here is the wording for the exception for spacing:

> Spacing: Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the [bounding box](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-minimum-bounding-box) of each, the circles do not intersect another target or the circle for another undersized target;

Each circle is going to extend a 12 pixel diameter from its center. With a 15px target, that's 7.5px. With 9px of padding, that's 4.5 pixels of padding in each direction. 7.5+4.5=12 pixels. Although the circles are not going to abut, they are not going to intersect. The menu passes.

**Comment by ljoakley** [comment]
Thanks for the explanation.
